api/envoy/type/http_status.proto

```diff
@@ -94,6 +94,8 @@ enum StatusCode {
 
   ExpectationFailed = 417;
 
+  ImATeapot = 418;
```

Comment:
Only need to add to v3, as we've frozen v2.

Reply:
@htuch Updated!